lib/validators/constants.rb

```diff
@@ -1,5 +1,6 @@
 module Validators
-  EMAIL_FORMAT = /\A[a-z0-9]+([-._][a-z0-9]+)*(\+[^@]+)?@[a-z0-9]+([.-][a-z0-9]+)*\.[a-z]{2,4}\z/i
+  # Support long suffix domain, like: .museum, .fitness
+  EMAIL_FORMAT = /\A[a-z0-9]+([-._][a-z0-9]+)*(\+[^@]+)?@[a-z0-9]+([.-][a-z0-9]+)*\.[a-z]{2,}\z/i
```

Comment:
And `tld: true` works only for hostname validator.

I agree about to use strict rules, but IMO, long suffix domains is required about this rules today. So, to be strict about long suffix domains, or we change this basic rule `/...\.[a-z]{2,4}\z/i` to `...\.(?:#{tld_long_suffix_domains_list.join("|")})\z/i`, or apply the `tld` options for email validator. Even for this last one, we need to be less strict on domain suffix validation.